### ISSUE TITLE
`emacs-jupyter` forces white background when inserting into org-mode buffers

### ISSUE THREAD

**Issue by rprechelt** [dzop/emacs-jupyter#349]
I have my `matplotlibrc` file setup to save `.png` (by default) with `transparent` backgrounds. If I do, 

``` python
import matplotlib.pyplot as plt
plt.plot([1, 2, 3])
plt.savefig("image.png")
```
in a web Jupyter notebook, I get a transparent plot. This is the same result in both browser-based Jupyter notebooks and the Jupyter CLI.

If I create a figure directly in an `emacs-jupyter` session, the preview that is shown _in the buffer has an opaque white background_ but if I _manually_ save it to disk _before_ I display it into the buffer, then it will be transparent. 

If I run the _exact same code_ in a `jupyter-python` code block in an org-mode buffer, the plot that is saved to disk _and_ the preview that is shown in the buffer has a _white_ background. 

Even if I _manually_ force the particular plot to be transparent in the code block (with `ax.set_facecolor()` and `fig.set_facecolor`), the_ resulting `.png` and preview in the buffer _will still be_ opaque with a white background. 

Basically, if emacs ever tries to display the resulting plot into a buffer, then it appears to be forcing a white background (both in the buffer _and_ in the file saved into `.ob-jupyter/`), otherwise it honors my `matplotlibrc` and code blocks.

I've tried to take a whirl through the repo but I can't see anything obvious. If it was only opaque in the emacs buffer but not on disk, I thought it might be related to some encoding that was throwing away the alpha channel but it _also makes the plots saved to disk opaque_. 

Do you have any ideas? Can you point me to the code where plots are recognized and saved to disk from an org-mode source block?

**Comment by nnicandro** [comment]
The function that is used to save an image to disk is `jupyter-org--image-result` which just writes the image data as returned by the kernel to disk and returns an Org image link to be inserted into the buffer. `emacs-jupyter` doesn't mess with the image display stuff, it just writes the image to disk and lets Org handle the display of the link to the written image file.
